### DISCUSSION TITLE
Not Plotting Maps Properly

### DISCUSSION THREAD

**Discussion by ve3hls** [magicbug/Cloudlog#1054] [General]
I'm trying to plot a map of the stations I worked during the CQWW WPX CW contest this past weekend.
In Custom Maps I pull down the calendars to enter the start and finish dates and times.  My settings have time set up as UTC so I am entering the times in UTC.

The resulting map shows only a small fraction of the stations I worked.  It's as if I enter the time in UTC but it assumes another time zone.  When I first start the Custom Maps app (for lack of a better description) I can see that it's pulling the time from my computer, which is set to UTC, but it's still not displaying all of the QSOs made in that period.

73,
Ken, VE3HLS

**Comment by magicbug** [comment]
Can confirm this is a bug.

**Comment by magicbug** [comment]
Issue ticket created at https://github.com/magicbug/Cloudlog/issues/1055 will get it fixed soon as I can

**Comment by ve3hls** [comment]
Thanks Peter!
Much appreciated.

Ken,
So Phisai, Thailand
Blog:  bueng-ken.com
